packages/react-native/Libraries/Text/Text/RCTTextView.mm

```diff
@@ -207,6 +207,22 @@ - (void)setTextStorage:(NSTextStorage *)textStorage
   [self setNeedsDisplay];
 }
 
+#if TARGET_OS_OSX // [macOS
+- (NSRect)getRectForCharRange:(NSRange)charRange
```

Comment:
nit: rename to `rectForCharRange:`, based on [Apple's naming guidelines](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html)